Explorer/Assets/DCL/Editor/EntityBrowserWindow.cs

```diff
@@ -0,0 +1,393 @@
+using Arch.Core;
+using Arch.Core.Utils;
+using Global.Dynamic;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Pool;
+using Object = UnityEngine.Object;
+
+namespace DCL.Editor
+{
+    public sealed class EntityBrowserWindow : EditorWindow
+    {
+        private float scrollY;
+        private static readonly GUILayoutOption[] SINGLE_LINE_HEIGHT_LAYOUT
+            = { GUILayout.Height(EditorGUIUtility.singleLineHeight) };
+
+        // Query editor
+        private QueryDescription query = new ();
+        private readonly HashSet<Type> all = new ();
+        private readonly HashSet<Type> any = new ();
+        private readonly HashSet<Type> none = new ();
+        private readonly HashSet<Type> exclusive = new ();
+        private HashSet<Type> unfoldedQueryTerm;
+        private Type[] registryTypes;
+        private int registryTypeCount;
+
+        // Entity list
+        private readonly Dictionary<(int, int), ComponentUIState> componentUiStates = new ();
+        private readonly Dictionary<int, EntityUIState> entityUiStates = new ();
+        private readonly HashSet<int> existingEntities = new ();
+        private static readonly Dictionary<Type, (FieldInfo, string)[]> FIELD_CACHE = new ();
+
+        [MenuItem("Arch/View/Entities")]
+        private static void ShowWindow()
+        {
+            var window = EditorWindow.GetWindow<EntityBrowserWindow>("Entities");
+            window.Show();
+        }
+
+        private void OnGUI()
+        {
+            scrollY = EditorGUILayout.BeginScrollView(new Vector2(0f, scrollY)).y;
+
+            // World selector
+
+            EditorGUI.BeginDisabledGroup(true);
+            EditorGUILayout.Popup("World", 0, new[] { "Global" });
```

Comment:
Have you considered to extend `SystemGroupSnapshot` to include scene worlds? The only missing thing there is the reference to the world itself. Thus, you will be able to select them here

Reply:
I did not find a way to do it in one minute, so I gave up because I did not need that at the moment. I'll take a look at SystemGroupSnapshot. Overall, there is no definite end to this tool, so I have to stop at some point and say "this is good enough for now".

Reply:
I can't find where scene worlds are.